server/src/test/java/org/spine3/server/entity/storage/EntityQueriesShould.java

```diff
@@ -91,27 +93,25 @@ public void construct_non_empty_queries() {
         final BoolValue archivedValue = BoolValue.newBuilder()
                                                  .setValue(true)
                                                  .build();
+        final ColumnFilter versionFilter = ColumnFilters.eq("version", versionValue);
```

Comment:
I would have an enum in the production code for the `version` and `archived` columns. And use the enum values here.